src/libs/actions/Report.ts

```diff
@@ -1972,6 +1972,12 @@ function shouldShowReportActionNotification(reportID: string, action: ReportActi
         return false;
     }
 
+    // If this is a whisper targeted to someone else, don't show it
+    if (action && ReportActionsUtils.isWhisperActionTargetedToOthers(action)) {
```

Comment:
It might be clearer if the condition would get for whispers to you only and return true but NAB

Reply:
ah yes, but this requires more changes, I'll open a refactor PR that will be properly tested/QA'ed.